src/java/voldemort/VoldemortClientShell.java

```diff
@@ -117,7 +117,7 @@ public VoldemortClientShell(ClientConfig clientConfig,
         try {
             factory = new SocketStoreClientFactory(clientConfig);
             client = factory.getStoreClient(storeName);
-            adminClient = new AdminClient(clientConfig);
+            adminClient = new AdminClient(clientConfig.setIdentifierString("admin"));
```

Comment:
You get an ugly exception when closing the Voldemort client shell but nothing more.